projects/plugins/jetpack/extensions/blocks/dialogue/editor.scss

```diff
@@ -42,3 +41,17 @@
 .wp-block-jetpack-dialogue__participant.is-editing-participant {
 	opacity: 0.7;
 }
+
+.wp-block-jetpack-conversation:not(.is-style-column) {
+	.wp-block-jetpack-dialogue__meta {
+		// The div below is added by the withFocusOutside HoC.
+		// We need to stretch it out to deal with speaker width.
+		&.has-not-media-source > div:not([class]) {
```

Comment:
Would it simplify this to leave out the `:not([class])` selector? The `div` it's targeting is the only `div` that is a direct descendent of the `.has-not-media-source` one, so should be handled with the `> div`.

Reply:
yes, I wasn't sure to apply this rule just to the straight children but I think you're right. Let me update it.

Reply:
addressed here https://github.com/Automattic/jetpack/pull/18832/commits/09707301e41f44739d638928d3211d2f95f209c4